src/templates_front/templator_front.org

```diff
@@ -83,7 +83,8 @@ module trexio
   implicit none
 
   integer, parameter :: trexio_exit_code = c_int32_t
-  integer, parameter :: trexio_backend = c_int32_t
+  integer, parameter :: trexio_backend   = c_int32_t
+  integer, parameter :: trexio_filetype  = c_int64_t
```

Comment:
General question here: should the names of these types be consistent with the C API ?

For example, we have `trexio_exit_code` in both C and Fortran, but then we have `back_end_t` and `trexio_t` in C versus `trexio_backend` and `trexio_filetype` in Fortran.

Reply:
Correct!
In the C we should have `trexio_back_end_t` instead of `back_end_t`